indices/utils.py

```diff
@@ -151,4 +153,148 @@ def parse_mesh_headings(metadata_dir: str,
         dois = set(article_df['doi'])
         heading_to_dois[heading] = dois
 
-    return heading_to_dois
\ No newline at end of file
+    return heading_to_dois
+
+
+def calculate_percentiles(true_vals, doi_to_shuffled_metrics):
```

Comment:
If I remember correctly some of these functions are in your notebooks too, right? Would it be a lot of work to have a single version of them and call the same functions from the web app and the notebook, or is it not worth it to reorganize your project structure that way?

Reply:
Right, the functions originally lived in the notebooks, then I copied them over when the web app was its own directory, then I moved them into `utils` when merged the two directories. I will add an issue to use the `utils` version of the functions in whichever notebooks I end up keeping